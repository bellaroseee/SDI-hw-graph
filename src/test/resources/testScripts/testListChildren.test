# Test Printing outgoing edges from a node

# Create a graph
CreateGraph graph1

# Add 2 pair of nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add 2 edges
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n3 e2

# Print the outgoing edges from n1
ListChildren graph1 n1

# Add another 2 pair of nodes
AddNode graph1 n4
AddNode graph1 n5
AddNode graph1 n6

# Add another 2 edges
AddEdge graph1 n4 n5 e3
AddEdge graph1 n4 n6 e4

# Print the outgoing edges from n1
ListChildren graph1 n1
ListChildren graph1 n4