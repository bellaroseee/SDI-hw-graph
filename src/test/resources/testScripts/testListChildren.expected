# Test Printing outgoing edges from a node

# Create a graph
created graph graph1

# Add 2 pair of nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add 2 edges
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n3 in graph1

# Print the outgoing edges from n1
the children of n1 in graph1 are: n2(e1) n3(e2)

# Add another 2 pair of nodes
added node n4 to graph1
added node n5 to graph1
added node n6 to graph1

# Add another 2 edges
added edge e3 from n4 to n5 in graph1
added edge e4 from n4 to n6 in graph1

# Print the outgoing edges from n1
the children of n1 in graph1 are: n2(e1) n3(e2)
the children of n4 in graph1 are: n5(e3) n6(e4)